src/inputcard.ts

```diff
@@ -73,6 +74,28 @@ document.addEventListener('DOMContentLoaded', () => {
   // init modals
   modalWindows();
 
+  // get and set terms of services
+  async function setTermOfService() {
+    await TE.tryCatch(() => pmClient.getResourcesUsingGET({ language: 'it' }), toError)
+      .fold(
+        () => undefined, // to be replaced with logic to handle failures
+        myResExt => {
+          const termini = myResExt.fold(
+            () => 'notFound :(',
+            myRes => (myRes.status === 200 ? myRes.value?.data?.termsAndConditions : 'notFound :('),
+          );
+          const termsAndService = modalAndTerm?.getElementsByClassName('modalwindow__content')[0];
```

Comment:
TIP: If you want to keep the first element found by a query to the DOM you could use ["querySelector"](https://developer.mozilla.org/it/docs/Web/API/Document/querySelector) instead of "getElementsByClassName, which returns the first element encountered ;-)

Reply:
Replaced ! Tnx for the tip : )